CMakeLists.txt

```diff
@@ -22,6 +22,29 @@ endif()
 
 project(polybar C CXX)
 
+# Polybar version information
+# Update this on every release
+# This is used to create the version string if a git repo is not available
+set(VERSION_MAJOR "3")
+set(VERSION_MINOR "3")
+set(VERSION_PATCH "0")
```

Comment:
Instead of writing this out manually, we could do some parsing of `version.hpp` like
```
file(READ "include/version.hpp" ver)
string(REGEX MATCH "...")
```
similar to [this](https://stackoverflow.com/a/47084079).

Reply:
We actually remove the `version.hpp` in this PR.

Wouldn't parsing just create more problems? Or is there some benefit to your suggestion that I'm not seeing?

Reply:
I prefer having a separate file just for versioning to make things simpler to read/find, but it shouldn't matter too much.

Reply:
Fair enough, shouldn't be too difficult to add a `version.txt` or something like that.

Reply:
Done